v2/googlecloud-to-googlecloud/src/main/java/com/google/cloud/teleport/v2/options/SpannerChangeStreamsToGcsOptions.java

```diff
@@ -175,9 +161,7 @@ public interface SpannerChangeStreamsToGcsOptions
       optional = true,
       description = "Window duration",
       helpText =
-          "The window duration/size in which data will be written to Cloud Storage. Allowed formats"
-              + " are: Ns (for seconds, example: 5s), Nm (for minutes, example: 12m), Nh (for"
-              + " hours, example: 2h).",
+          "The window duration is the interval in which data is written to the output directory. Configure the duration based on the pipeline's throughput. For example, a higher throughput might require smaller window sizes so that the data fits into memory. Defaults to 5m, with a minimum of 1s. Allowed formats are: [int]s (for seconds, example: 5s), [int]m (for minutes, example: 12m), [int]h (for hours, example: 2h).",
```

Comment:
```suggestion
          "The window duration is the interval in which data is written to the output directory. Configure the duration based on the pipeline's throughput. For example, a higher throughput might require smaller window sizes so that the data fits into memory. Defaults to 5m (five minutes), with a minimum of 1s (one second). Allowed formats are: [int]s (for seconds, example: 5s), [int]m (for minutes, example: 12m), [int]h (for hours, example: 2h).",
```